main

```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/env ruby
+
+require 'pry'
+require './books'
+require './classroom'
+require './corrector'
+require './person'
+require './rental'
+require './students'
+require './teacher'
+
+NEW_LINE = "\n".freeze
+
+$person_list = []
+$book_list = []
+$rentals = []
+
+def menu
```

Comment:
- [ ] On the top of every _options menu_ the text `Please choose an option by entering a number:` should show

**Your UI:**
![image](https://user-images.githubusercontent.com/12745474/141559613-3532b3a8-e4c2-4870-a6e8-5d8d824d778f.png)

**The original UI:**
![image](https://user-images.githubusercontent.com/12745474/141561049-b4cfcda3-e975-451f-b8f1-428ce57a86ab.png)

Reply:
This issue is well resolved! ✔️